bom/pom.xml

```diff
@@ -162,8 +162,18 @@
             </dependency>
 
 
+
+            <!-- Windup Rules Base -->
+            <dependency>
```

Comment:
Since this PR was rejected, I replicated this in https://github.com/windup/windup/pull/428